src/components/form.scss

```diff
@@ -43,45 +121,57 @@
 
 .form-label {
   color: var(--op-color-on-background);
```

Comment:
@Jeremy-Walton here it looks like were making the highest possible contrast choice since we don't know where the control will be used. This can be changed if needed per project so this seems like the safest assumption to make. Was this you're thinking as well?

Reply:
Correct.